src/h5web/visualizer/Visualizer.test.tsx

```diff
@@ -155,5 +169,8 @@ describe('Visualizer', () => {
 
     await selectExplorerNode('signal_not_found');
     expect(await screen.findByText(/to exist/u)).toBeVisible();
+
+    expect(consoleMock).toHaveBeenCalledTimes(8); // React logs two stack traces per error
+    resetConsole();
```

Comment:
The errors thrown in `supportsEntity` are now caught by the `ErrorBoundary` in `App`, which means that errors are now logged to the console.

Reply:
Well, I do: I test the presence of the error message on screen displayed by the `ErrorBoundary`.

Reply:
By the way, the great thing about the way we test is that all of these `expect(await screen.findByText(/to exist/u)).toBeVisible();` did not actually break even though we now display errors with an `ErrorBoundary` instead of rendering them ourselves!

Reply:
Ah yes indeed sorry :slightly_frowning_face: